app/assets/stylesheets/admin/loans/_tables.scss

```diff
@@ -19,7 +19,7 @@ table.breakeven-table-rampup {
   }
 }
 
-.editable-table {
+.editable-table, .editable-breakeven-table {
```

Comment:
The main reason for the separate class is for the breakeven table to attach the backbone view to. I opted for this, instead of using a not() selector in loan_questionnaires_view.